src/commands/cmd_dispatcher.c

```diff
@@ -66,6 +66,11 @@ int CommandDispatch(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
 	if(_validate_command_arity(cmd, argc) == false) return RedisModule_WrongArity(ctx);
 	Command_Handler handler = get_command_handler(cmd);
 	GraphContext *gc = GraphContext_Retrieve(ctx, graph_name, true, true);
+	// Check if graph is in decode - The graph key has been decoded but not all the virtual keys finished.
+	if(GraphContext_IsInDecode(gc)) {
```

Comment:
`gc` will be NULL if the key is already set as a non-graph value, causing a crash here. `GraphContext_Retrieve` should probably have handling for this scenario?

Reply:
cmd_dispatcher call `GraphContext_Retrieve` with the `shouldCreate` flag set to `true` so it will never return null.

Reply:
```
127.0.0.1:6379> SET G 5
OK
127.0.0.1:6379> GRAPH.QUERY G "MATCH (a) RETURN a"
Could not connect to Redis at 127.0.0.1:6379: Connection refused
```
```
Thread 1 "redis-server" received signal SIGSEGV, Segmentation fault.
0x00007ffff6434e17 in GraphContext_IsInDecode (gc=0x0) at src/graph/graphcontext.c:136
136             return GraphDecodeContext_IsGraphInDecode(gc->decoding_context);
```

Strictly speaking, I believe this bug exists on master too, so it can be considered outside of this PR's scope.